packages/framework/presence/src/presenceManager.ts

```diff
@@ -85,6 +85,12 @@ class PresenceManager
 
 		runtime.on("connected", this.onConnect.bind(this));
 
+		runtime.on("disconnected", () => {
+			if (runtime.clientId !== undefined) {
+				this.removeClientConnectionId(runtime.clientId);
+			}
+		});
```

Comment:
We can keep this runtime dependency out of here and place it in `PresenceManagerDataObject` where the existing call to `removeClientConnectionId` is. (We want to keep the runtime dependencies to a minimum.) I assume from this that audience never says "removeMember" for self.

Reply:
I think I made the wrong comment above. It is cool to have the call here (perhaps better at least for now). The more important thing is consistency. We can move the `PresenceManagerDataObject` call to here since audience is being added to IEmphemeralRuntime (in #23006) and remove `ClientConnectionManager` that requires `removeClientConnectionId` to be public. Related comment from #22833: https://github.com/microsoft/FluidFramework/pull/22833#discussion_r1814229495